Bottom: db445c455057048f37af8c742ba894df330acb88
Top:    8a8bc44c8b4a62fb4372b3350a8641c4ff411d1f
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 19:43:23 +0800

Refresh of tun-introduce-0

---

diff --git a/include/linux/if_tun.h b/include/linux/if_tun.h
index 2e45091..bf9bdf4 100644
--- a/include/linux/if_tun.h
+++ b/include/linux/if_tun.h
@@ -19,7 +19,7 @@
 
 #if defined(CONFIG_TUN) || defined(CONFIG_TUN_MODULE)
 struct socket *tun_get_socket(struct file *);
-struct skb_array *tun_get_skb_array(struct file *);
+struct skb_array *tun_get_skb_array(struct file *file);
 #else
 #include <linux/err.h>
 #include <linux/errno.h>
